src/actuator/ActuatorSimple.C

```diff
@@ -0,0 +1,1204 @@
+    return vec;
+  return vec;  // Should not get here
+}
+
+std::vector<double> 
+ActuatorSimple::get_blade_area_elems(
+   std::vector<double> chord_table, 
+   const Coordinates &p1,  
+   const Coordinates &p2,
+   const int &Npts)
+{
+  // stk::mesh::MetaData& metaData = realm_.meta_data();
+  // const int nDim = metaData.spatial_dimension();
+  const int nDim = 3;
+  std::vector<double> areas(Npts, 0.0);
+  double denom = (double)Npts; //Npts - 1.0;
+  std::vector<double> dx(nDim, 0.0);
+
+  dx[0] = (p2.x_ - p1.x_)/denom; 
+  dx[1] = (p2.y_ - p1.y_)/denom; 
+  if (nDim>2) dx[2] = (p2.z_ - p1.z_)/denom; 
+  // Assumes equal area spacing
+  double dx_norm = sqrt(dx[0]*dx[0] + dx[1]*dx[1] + dx[2]*dx[2]);
+  for (int i=0; i<chord_table.size(); i++) {
+    areas[i] = dx_norm*chord_table[i];
+  }
+  
+  return areas;
+}
+
+
+//--------------------------------------------------------------------------
+//--------------------------------------------------------------------------
+//--------------------------------------------------------------------------
+//-------- compute_elem_centroid -------------------------------------------
+//--------------------------------------------------------------------------
+void
+ActuatorSimple::compute_elem_centroid(
+  const int& nDim, double* elemCentroid, const int& nodesPerElement)
+{
+  // zero
+  for (int j = 0; j < nDim; ++j)
+    elemCentroid[j] = 0.0;
+
+  // assemble
+  for (int ni = 0; ni < nodesPerElement; ++ni) {
+    for (int j = 0; j < nDim; ++j) {
+      elemCentroid[j] += ws_coordinates_[ni * nDim + j] / nodesPerElement;
+    }
+  }
+}
+
+// Spread actuator force to nodes
+void
+ActuatorSimple::spread_actuator_force_to_node_vec(
+  const int& nDim,
+  std::set<stk::mesh::Entity>& nodeVec,
+  const std::vector<double>& actuator_force,
+  // The tensor to indicate the orientation of the airfoil sections
+  const std::vector<double>& orientation_tensor,
+  const double* actuator_node_coordinates,
+  const stk::mesh::FieldBase& coordinates,
+  stk::mesh::FieldBase& actuator_source,
+  const stk::mesh::FieldBase& dual_nodal_volume,
+  const Coordinates& epsilon,
+  const std::vector<double>& hubPt,
+  const std::vector<double>& hubShftDir,
+  std::vector<double>& thr,
+  std::vector<double>& tor)
+{
+
+  std::vector<double> ws_nodeForce(nDim);
+
+  // This is the distance vector
+  std::vector<double> distance(nDim);
+  // This is the distance vector projected onto the blade coordinate system
+  std::vector<double> distance_projected(nDim, 0.0);
+
+  // Loop through all the mesh points influenced by the body force
+  std::set<stk::mesh::Entity>::iterator iNode;
+  for (iNode = nodeVec.begin(); iNode != nodeVec.end(); ++iNode) {
+
+    stk::mesh::Entity node = *iNode;
+
+////////////////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////
+///////////                Error is HERE
```

Comment:
Hopefully this comment is out-of-date

Reply:
yeah this is a left over from tony's dev process.  I deleted it in the actuator code. Please delete here too

Reply:
yes, those error statements are outdated, I missed them during my pull request some time ago. Please feel free to delete them.

Reply:
Deleted